src/api/app/models/staging/staged_requests.rb

```diff
@@ -81,6 +90,10 @@ def not_found_requests
     request_numbers - requests.pluck(:number).map(&:to_s)
   end
 
+  def excluded_requests
```

Comment:
I find this a very unfortunate function name taking that it does not return requests, but strings of numbers. And I find it unfortunate to call such an expensive query in a loop - is this codeclimate speaking? :)

Reply:
You are right. On it

Reply:
Fixed, check it out, please.